extractor/src/consumer_class.py

```diff
@@ -49,6 +48,7 @@ def _generate_config(self):
             "bootstrap.servers": self.broker,
             "group.id": self.group_id,
             "session.timeout.ms": 6000,
+            'metadata.max.age.ms': 5000,
```

Comment:
Why did you add that? Did you also add it in the other consumers?

Reply:
It's for refreshing the list of topics consumed (if we run new batch). So every 5s (default is 5 min). I will add a comment.